### PR TITLE
Explain how works *.forwarded sockets.

### PR DESCRIPTION
**Pull request by TSnake41**
Some daemons (xcp-rrdd, xenopsd, others ?) uses .forwarded sockets, but it is explained nowhere how they works and interact with the rest of XAPI world.

Add a small documentation on those sockets for development purposes.

### CODE REVIEW

**Comment by psafont** [comment]
Thanks for the contribution!

The page as it is wouldn't be linked from anywhere, could you add a link to the navigation bar? It should be as easy as adding it to this list: https://github.com/xapi-project/xapi-project.github.io/blob/master/_data/navbar.yml#L77

Because xapi is a linux project, there are check to require that the commits are signed by the committers as a way to state that they are the authors or they have permission from the authors.
This can be done for new commits with `git commit -s`, and can be added to your existing commit with `git commit --amend -s`

File: devguide/howtos/use-forwarded.socket.md
Comment (lindig) [inline devguide/howtos/use-forwarded.socket.md:11]: Please warp lines to facilitate review here. (Not a problem in this specific case but in general MD files should not exceed 80 characters per line.

File: devguide/howtos/use-forwarded.socket.md
Comment (lindig) [inline devguide/howtos/use-forwarded.socket.md:37]: Please run through a pretty printer that also wraps and indents lines.

**Review by lindig — APPROVED** [review]

File: devguide/howtos/use-forwarded.socket.md
Comment (julien-f) [reply devguide/howtos/use-forwarded.socket.md]: Don't know if that applies here, but in the Xen Orchestra project, we are using [Prettier](https://prettier.io/) to format all files, including Markdown :slightly_smiling_face:

Comment (lindig) [reply devguide/howtos/use-forwarded.socket.md]: I didn't know that this exists for Markdown and will explore it. We don't do It by default.

**Comment by lindig** [comment]
I suggest we merge this

**Review by psafont — APPROVED** [review]

**Comment by lindig** [comment]
Does this need a rebase? "This branch cannot be rebased due to too many changes"; I will merge this once a clean merge is possible @TSnake41

**Comment by psafont** [comment]
This is because new commits get added to the master branch whenever xen-api changes its API

**Pull request merged**
